test/cli/license_test.go

```diff
@@ -3,7 +3,7 @@ package cli
 import "testing"
 
 func Test_Licenses(t *testing.T) {
-	testImage := getFixtureImage(t, "image-unknown-licenses")
```

Comment:
it looks like this test fixture is no longer used, so we should remove it

Reply:
https://github.com/anchore/syft/pull/3901